besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -3672,4 +3672,17 @@ public void assertThatInvalidWsTimeoutSecondsFails() {
     assertThat(commandErrorOutput.toString())
         .contains("Invalid value for option", "--Xws-timeout-seconds", "abc", "is not a long");
   }
+
+  @Test
```

Comment:
Two more tests - no ports specified and every possible port specified - p2pPort, graphQLHttpPort, rpcHttpPort, rpcWsPort, metricsPort, metricsPushPort, stratumPort